neuraxle/pipeline.py

```diff
@@ -131,18 +131,18 @@ def load(self, pipeline: 'Pipeline', data_container: DataContainer) -> 'Pipeline
         pipeline_cache_folder = os.path.join(self.cache_folder, pipeline.name)
         pipeline_cache_list_file_name_path = os.path.join(pipeline_cache_folder, self.pipeline_cache_list_file_name)
         if not os.path.exists(pipeline_cache_list_file_name_path):
-            return pipeline
+            return None
```

Comment:
Smell: no null returns. 

Fix: add a `can_load` method as already suggested in another comment of this pr (see more above for the 1st commit's comments). 

This is an idea, I'm not sure if it's needed to fix this. For instance, with the data checkpoints this was needed to avoid disk I/O operations when not required.

Reply:
EDIT: I saw before the `try_...` method. I prefer with the fix I suggest. The `can_load` method should be performant and just have to check if the file name exists (e.g.: hash file name with byte code's hash and perhaps other hashes).